config/routes.rb

```diff
@@ -86,6 +89,9 @@
     end
   end
 
+  authenticate :user, lambda { |u| u.admin? } do
```

Comment:
> undefined method `authenticate' for #<ActionDispatch::Routing::Mapper

This is not valid syntax, You are not using Devise.

Reply:
Whatever @frett says. It would be nice if you could leverage Okta login and the User#admin boolean you already have, but there is no Web front end so that may be more trouble than it's worth.

Reply:
I agree it would be a nice tie in, but I think the cost would greatly outweigh the benefit. Let's use basic auth that checks user & pass against a couple environment variables.

```
  Sidekiq::Web.use Rack::Auth::Basic do |username, password|
    username == ENV.fetch("SIDEKIQ_USERNAME") && password == ENV.fetch("SIDEKIQ_PASSWORD")
  end
```

Reply:
@knutsenm right.. ok